csireverseproxy/pkg/standaloneproxy/standaloneproxy.go

```diff
@@ -109,7 +107,7 @@ func newReverseProxy(mgmtServer config.ManagementServer) common.Proxy {
 func newTLSConfig(mgmtServer config.ManagementServer) *tls.Config {
 	// #nosec G402
 	tlsConfig := tls.Config{
-		InsecureSkipVerify: mgmtServer.SkipCertificateValidation,
+		InsecureSkipVerify: mgmtServer.SkipCertificateValidation, // #nosec,G402
```

Comment:
can we leave a comment on why we are skipping cert validation here, and i assume telling gosec to skip the check as well.

Reply:
Added comment